code/game/machinery/camera/camera.dm

```diff
@@ -348,15 +342,12 @@
 	qdel(src)
 
 /obj/machinery/camera/update_icon() //TO-DO: Make panel open states, xray camera, and indicator lights overlays instead.
-	var/xray_module
-	if(isXRay(TRUE))
-		xray_module = "xray"
 	if(!status)
-		icon_state = "[xray_module][default_camera_icon]_off"
```

Comment:
wouldn't this require dmi changes?

Reply:
It would be null without an xray upgrade, so it will use the default camera icon instead which already exists.